src/main/java/org/opentripplanner/raptor/rangeraptor/standard/MinTravelDurationRoutingStrategy.java

```diff
@@ -182,10 +183,10 @@ private void logInvalidAlightTime(int stopPos, int stopArrivalTime) {
     if (logCount < 3) {
       ++logCount;
       LOG.error(
-        "Traveling back in time is not allowed. Board stop pos: {}, alight stop pos: {}, stop arrival time: {}, trip: {}.",
-        onTrip.findDepartureStopPosition(onTripBoardTime, onTripBoardStop),
+        "Traveling back in time is not allowed. Board time: {}, alight stop pos: {}, stop arrival time: {}, trip: {}.",
+        TimeUtils.timeToStrLong(onTripBoardTime),
         stopPos,
-        stopArrivalTime,
+        TimeUtils.timeToStrLong(stopArrivalTime),
         onTrip
       );
     }
```

Comment:
The `onTrip.findDepartureStopPosition(onTripBoardTime, onTripBoardStop)` is not working, we are searching in reverse here - so we should not use any of the trip methods inside Raptor - my mistake. Instead I print the boardTime - it is not as good, but the real importance here is to print the trip. I also added formatting to the times.